src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -668,6 +670,30 @@ def _flexible_server_params(command_group):
         with self.argument_context('{} flexible-server backup list'.format(command_group)) as c:
             c.argument('server_name', id_part=None, arg_type=server_name_arg_type)
 
+        # identity
+        if command_group == 'mysql':
+            with self.argument_context('{} flexible-server identity'.format(command_group)) as c:
+                c.argument('server_name', id_part=None, options_list=['--server-name', '-s'], arg_type=server_name_arg_type)
+
+            with self.argument_context('{} flexible-server identity assign'.format(command_group)) as c:
```

Comment:
Have you considered making an arg type for this parameter?

Reply:
Sounds good, done.